lib/peep.rb

```diff
@@ -0,0 +1,38 @@
+require 'PG'
+
+class Peep
+
+  attr_reader :id, :message, :time
+
+  def initialize(id:, message:, time:)
+    @id = id
+    @message = message
+    @time = time
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
```

Comment:
since you have the database_connection.rb the above if statement could be replaced by:
DatabaseConnection.query("SELECT * FROM peeps ORDER BY id DESC;")